### DISCUSSION TITLE
I couldn't configure it, help please

### DISCUSSION THREAD

**Discussion by marcelovidigal** [polymind-inc/acmebot#432] [Q&A] [answered]
I couldn't configure it, I tried a lot, the documentation is simple, it seems straightforward, but then you get stuck with an error and you don't know what alternatives you have. I couldn't solve these errors, the access policy is there, the DNS manager is from google, I made the respective configuration, but no certificate appears in the key vault or I always come across this error message: The activity function 'GetAllCertificates' failed : "The user, group or application ... does not have certificates list permission on key vault ... but, reinforcing, the access policies are there, I checked. I don't know what else to do, I ended up giving up. What could this be? Does anyone know another way to get lets encrypt certificates in Azure? This is a linux hosting and ends up having less options to automate lets encrypt in azure.

**Comment by shibayan** [comment]
As per the error message, you need to check again if there is any problem in setting the access permissions to Key Vault.

If you create Key Vault at the same time when deploying to Azure, you should also consider redeploying it as RBAC configuration is done automatically.

**Comment by Qvakk** [comment, marked as answer]
Are you sure you have the List-permission added to for the app in the accesspolicy?
example:
![image](https://user-images.githubusercontent.com/6873953/147982821-967beedf-1db7-4172-b221-b62e209bd149.png)
